doc/cephfs/nfs.rst

```diff
@@ -70,12 +70,17 @@ to get the benefit of sessions.
 Conventions for mounting NFS resources are platform-specific. The
 following conventions work on Linux and some Unix platforms:
 
-From the command line::
+.. code::bash
 
-  mount -t nfs -o nfsvers=4.1,proto=tcp <ganesha-host-name>:<ganesha-pseudo-path> <mount-point>
+    mount -t nfs -o nfsvers=4.1,proto=tcp <ganesha-host-name>:<ganesha-pseudo-path> <mount-point>
```

Comment:
```suggestion
 mount -t nfs -o nfsvers=4.1,proto=tcp <ganesha-host-name>:<ganesha-pseudo-path> <mount-point>
```

Reply:
Get rid of the indentation? Code blocks are required to be indented in case of RST flles, right?

Reply:
Yes. But I don't see the command on rendered doc https://ceph--38763.org.readthedocs.build/en/38763/cephfs/nfs/#mount-using-nfsv4-clients

Reply:
give a space `.. code:: bash`

Reply:
Thanks, Jos.